src/util/amount.rs

```diff
@@ -437,12 +451,12 @@ impl fmt::Debug for Amount {
     }
 }
 
-// No one should depend on a binding contract for Display for this type.
-// Just using Bitcoin denominated string.
+// No one should depend on a binding contract for Display for this type, however
+// make sats the standard.
 impl fmt::Display for Amount {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        self.fmt_value_in(f, Denomination::Bitcoin)?;
-        write!(f, " {}", Denomination::Bitcoin)
+        self.fmt_value_in(f, Denomination::Sats)?;
+        write!(f, " {}", Denomination::Sats)
```

Comment:
I think this will conflict with my PR. Is there a preferred order of merging?

Reply:
Do yours first man, I'm the new guy I'll work around you.